### DISCUSSION TITLE
Where I found the "Why Adonis" page ???

### DISCUSSION THREAD

**Discussion by rbgfloripa** [adonisjs/core#2597] [General]
Hi all,

Where is "Why Adonis" page ?
That page was good for showing other people the reasons for adopting adonis in new projects.

https://preview.adonisjs.com/why-adonisjs

**Comment by mastermunj** [comment]
Is [this](https://github.com/adonisjs/legacy.adonisjs.com/blob/develop/docs/pages/why-adonisjs.md) what you are looking for?

**Reply by rbgfloripa** [reply]
Thanks man !!
